### PR TITLE
Tool changes for Mac OS/X.

### PR DESCRIPTION
**Pull request by paulidale**
Perl ends up with a different path when installed using _brew_.  The system perl
seems to have broken _cpan_ somehow.

Some additional packages need to be installed.

PATH also needs to be set so it can find the _plackup_ executable.

### CODE REVIEW

File: OpenSSL-Query/README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ OpenSSL::Query requires these extra modules to run:
 - Class::Method::Modifiers	(debian package libclass-method-modifiers-perl)
 - Moo				(debian package libmoo-perl)
 - URI::Encode			(debian package liburi-encode-perl)
+- LWP::UserAgent
+- LWP::Protocol::https
```
Comment (levitte) [inline OpenSSL-Query/README.md]: Please change the "requires" section in `Makefile.PL` as well

Comment (paulidale) [reply OpenSSL-Query/README.md]: Added.
I'm not au fait with Perl.

**Comment by paulidale** [comment]
PS: you shouldn't be working on a Sunday :(
Especially this early.

File: OpenSSL-Query/README.md (hunk 1 above)
Comment (paulidale) [reply OpenSSL-Query/README.md]: I've updated the required section (I think).

**Comment by levitte** [comment]
Merged.

1de20a3a49e1b4f6e7104b84b8fc5594b144712a add dependencies to Makefile.PL
cd0711f60154d5da0f9974349b897a2615934554 Tool changes for Mac OS/X.

**Pull request closed without merging**